### ISSUE TITLE
Cite does not work with Zotero 5

### ISSUE THREAD

**Issue by adeldaoud** [beloglazov/zotero-scholar-citations#32]
Hi,

I am reporting that Zotero scholar citations does not work with Zotero 5, unfortunately. The Add-ons manager reports incompatibility with the new version. 

Can I tweak some parameter to make it work?

cheers

**Comment by JordanMagnuson** [comment]
Confirming that it's not working with Zotero 5. I was able to install the plugin, but my "Extra" field is being populated by garbage data: https://i.imgur.com/qA16ojK.png

**Comment by JordanMagnuson** [comment]
Never mind: it's working for me!

**Comment by hugo921** [comment]
@JordanMagnuson i have the same problem.how did you sovled it?

**Comment by JordanMagnuson** [comment]
The screenshot I originally shared had "extra" data that was populated by Zotero automatically when grabbing an item from Google Books; I didn't realize what was going on there, and so was confused.

When I add new items, though, this plugin works as expected by putting the google scholar citation count in front of anything else in the "extra" field, which allows sorting to work successfully. See screenshot: https://i.imgur.com/foTGVGr.png

So in summary: I didn't do anything--the plugin was working all along. 

**Except: I haven't yet seen it do automatic updating of items I already had in my library. In order to see the citation count I need to add a new item, or "duplicate" an existing item.**

**Comment by hugo921** [comment]
thanks for your reply.citation in my zotero still cant works .my zotero version is 5.0.24,i think maybe need the author does more work.

**Comment by niklasholm** [comment]
I'm also unable to install the add-on; I got an error message that the add-on is incompatible with Zotero. My Zotero version is 5.0.24, add-on version 1.9.3.

**Comment by rkachurka** [comment]
Extra field is populated with 0000

**Comment by L-PC** [comment]
@JordanMagnuson Wich version of Zotero and the plugin are you using?

**Comment by rkachurka** [comment]
Started to populate Extra field with number of citations. No idea why. By the way, thanks for an awesome work
